frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -41,6 +41,9 @@ const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefini
   }
 
   if (networkType === 'cnv-bridge') {
+    // cnv-bridge should not define type on root
+    delete config.type;
```

Comment:
This is a nice way to handle this. We can always change it if we find that others tend not to define the type at the root, as well.